examples/rack_server.rb

```diff
@@ -4,7 +4,7 @@
 require 'polyphony/http'
```

Comment:
Something that was out of scope for this first exploration, is that `polyphony/http` no longer resolves as expected.

Thought it was better for you to decide what the requires should look like, since it's important for dev UX.

Reply:
Yeah I think that should be `require 'polyphony/http/server`.